test/scripts/no-jasmine-focus/invalid.describe.only.ts

```diff
@@ -0,0 +1,9 @@
+describe.only('test me', () => {
```

Comment:
I would change the directory name to `no-focus-test` instead of `no-jasmine-focus`